### DISCUSSION TITLE
is a new note necessary for annotations?

### DISCUSSION THREAD

**Discussion by jcesguerram** [RyotaUshio/obsidian-pdf-plus#85] [Q&A]
There's something I still don't get. When I copy a section of a pdf, do I necessarily need to paste it in a new note for it to appear in the backlinks section?
Could highlighted sections and comments appear in backlinks without having to paste them in a new note?

**Comment by RyotaUshio** [comment]
It need not be a new note. You can paste a link to a text selection to any markdown file in your vault, and then the highlight will show up in the PDF viewer.
